platform/ui/src/components/studyList/StudyList.js

```diff
@@ -208,6 +221,7 @@ StudyList.propTypes = {
   }).isRequired,
   onFilterChange: PropTypes.func.isRequired,
   studyListDateFilterNumDays: PropTypes.number,
+  displaySize: PropTypes.string,
```

Comment:
Can we do a value, oneOf for each of the options?

Reply:
should I? The idea was not to allow dev set any value he wants? Not necessary would be a value oneOf ['small','medium','large'].. Or should I restrict into it? If thats the case I would say to turn it into a app config and use from there

Reply:
You're correct. Sleepy brain was lying to me here. Thank you!